class-users-list.php

```diff
@@ -445,7 +447,7 @@ private static function get_credential_users() {
 		// phpcs:enable WordPress.Security.NonceVerification, WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
 
 		if ( is_array( $credential_users ) ) {
-			$users = array_map( 'absint', array_keys( $credential_users ) );
+			$users = array_map( 'absint',  $credential_users );
```

Comment:
`array_map` allows us to run a function against all members of an array. 
Since`$credential_users` is **not** an associative array but rather an indexed array, running `absint()` on the keys of the array(from `array_keys`) will yield an indexed array with incorrect ids. We should just run `absint()` against the values, to return an array of numbers.